src/cloudwatch/modules/client/ec2getclient.py

```diff
@@ -0,0 +1,82 @@
+import re
+
+from ..plugininfo import PLUGIN_NAME, PLUGIN_VERSION
```

Comment:
Same advice for imports, but see below for "does this module need to exist at all?"